Class-16-(ORM-Part-1).md

```diff
@@ -0,0 +1,31 @@
+# Class 16: Introduction to Object-Relational Mapping
+
+## In the prep work this week, the students learned
+
+1. What object-relational mapping is.
+1. How to connect a Spring application to a MySQL database.
+1. How to create an entity class.
+1. How to create a repository interface.
+
+## Announcements
+
+1. Assignment #4 is open and students can start working on it!
+1. Check with your course manager for any additional announcements.
+
+## Large Group Time (Instructor)
+
+### Class 16 Topics That Require Careful Attention
```

Comment:
I'd also add a note to review the relationships between Java classes/objects vs. database tables/rows.

Reply:
Done!